dirty_cat/test/test_docstrings.py

```diff
@@ -0,0 +1,172 @@
+import re
```

Comment:
A think that a module-level docstring giving a short description of what is in this module would be useful (for the future developers discovering the codebase)

Reply:
Okay, I added a small description, let me know what you think.

Reply:
I think there was a misunderstanding here: from what I understand, your description explains why we use `re`, but I think what Gaël meant was to clarify what this file (`test_docstrings`) does.

I'm thinking of something along the lines of *"This test suite ensures the docstrings of functions and classes in dirty_cat are formatted according to numpydoc specifications"*.